client/packages/host/src/components/Login/LoginIcon.tsx

```diff
@@ -11,8 +11,8 @@ export const LoginIcon = ({ small = false }: { small?: boolean }) => {
   const [customLogo] = useLocalStorage('/theme/logo');
   const isSmallScreen = useIsSmallScreen() || small;
   const logoStyle = isSmallScreen
-    ? { width: 61, height: 90 }
-    : { width: 122, height: 180 };
+    ? { width: 155, height: 90 }
```

Comment:
and an offroad.. to allow the PNG custom theme to have a logo that isn't squashed up. The normal logo looks fine, as well as the PNG.